code/src/components/AlbumContainer.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { AlbumCard } from "./AlbumCard";
+
+
+export const AlbumContainer = (props) => {
+  const { albums } = props;
+ 
+
+  return (
+    <article className="album-container">
+      {albums.map((album) => {
+        return (
+          <AlbumCard
+            key={album.id}
+            title={album.name}
+            artists={album.artists}
+            image={album.images[1].url}
+            albumLink={album.external_urls.spotify}
+            artistLink={album.external_urls.spotify}
```

Comment:
These two are the same. I'd say it's enough with the `albumLink `and then for the artists' URL, it's enough to have the `artists` so that we can map later on

Reply:
Yes, fixed it now, so the artist url is mapped in the Album card component!